cpp/dolfinx/io/cells.cpp

```diff
@@ -275,6 +275,74 @@ std::vector<std::uint8_t> gmsh_quadrilateral(int num_nodes)
     throw std::runtime_error("Higher order GMSH quadrilateral not supported");
   }
 }
+//-----------------------------------------------------------------------------
+std::vector<std::uint8_t> discontinuous_triangle(int num_nodes)
```

Comment:
Could this function name be more expressive?

Reply:
DG triangle, as it is directly related to the DG function space?

Reply:
No longer needed